provider/cluster_autoscaler_resource.go

```diff
@@ -0,0 +1,703 @@
+/*
+Copyright (c) 2023 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package provider
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/terraform-redhat/terraform-provider-rhcs/provider/common"
+)
+
+type ClusterAutoscalerResourceType struct {
+}
+
+type ClusterAutoscalerResource struct {
+	collection *cmv1.ClustersClient
+}
+
+func (t *ClusterAutoscalerResourceType) GetSchema(ctx context.Context) (
+	tfsdk.Schema, diag.Diagnostics) {
+
+	return tfsdk.Schema{
+		Description: "Cluster-wide autoscaling configuration.",
+		Attributes: map[string]tfsdk.Attribute{
+			"cluster": {
+				Description: "Identifier of the cluster.",
+				Type:        types.StringType,
+				Required:    true,
+				PlanModifiers: []tfsdk.AttributePlanModifier{
+					ValueCannotBeChangedModifier(),
+				},
+			},
+			"balance_similar_node_groups": {
+				Description: "Automatically identify node groups with " +
+					"the same instance type and the same set of labels and try " +
+					"to keep the respective sizes of those node groups balanced.",
+				Type:     types.BoolType,
+				Optional: true,
+			},
+			"skip_nodes_with_local_storage": {
+				Description: "If true cluster autoscaler will never delete " +
+					"nodes with pods with local storage, e.g. EmptyDir or HostPath. " +
+					"true by default at autoscaler.",
+				Type:     types.BoolType,
+				Optional: true,
+			},
+			"log_verbosity": {
+				Description: "Sets the autoscaler log level. " +
+					"Default value is 1, level 4 is recommended for DEBUGGING and " +
+					"level 6 will enable almost everything.",
+				Type:     types.Int64Type,
+				Optional: true,
+			},
```

Comment:
Can you add the default value? 
```suggestion
			"log_verbosity": {
				Description: "Sets the autoscaler log level. " +
					"Default value is 1, level 4 is recommended for DEBUGGING and " +
					"level 6 will enable almost everything.",
				Type:     types.Int64Type,
				Optional: true,
				Computed: true,
				Default:     int64default.StaticInt64(1)
			},
```